errors.go

```diff
@@ -35,7 +36,19 @@ var ErrMissingGuildName = fmt.Errorf("guild: %w", ErrMissingName)
 var ErrMissingChannelName = fmt.Errorf("channel: %w", ErrMissingName)
 var ErrMissingWebhookName = fmt.Errorf("webhook: %w", ErrMissingName)
 var ErrMissingThreadName = fmt.Errorf("thread: %w", ErrMissingName)
+var ErrMissingScheduledEventName = fmt.Errorf("scheduled event name: %w", ErrMissingName)
 
 var ErrMissingWebhookToken = errors.New("webhook token was not set")
 
 var ErrIllegalValue = errors.New("illegal value")
+var ErrIllegalScheduledEventPrivacyLevelValue = fmt.Errorf("scheduled event privacy level: %w", ErrIllegalValue)
+
+var ErrMissingTime = fmt.Errorf("time: %w", ErrMissingRequiredField)
+var ErrMissingScheduledEventStartTime = fmt.Errorf("scheduled event start: %w", ErrMissingTime)
+var ErrMissingScheduledEventEndTime = fmt.Errorf("scheduled event end: %w", ErrMissingTime)
+
+var ErrMissingLocation = fmt.Errorf("location: %w", ErrMissingRequiredField)
```

Comment:
What location is this?

Reply:
it seems unused. Perhaps just merge it into ErrMissingScheduledEventLocation? As Location in itself doesn't provide much context.

Reply:
yup, i just provide it for next development (if any)

Reply:
@andersfylling the code has been updated, please kindly check it again.